src/renderer/browser/util/parseURL.js

```diff
@@ -1,7 +1,16 @@
-import { trim } from 'lodash';
+import { filter, isEmpty, trim, trimEnd, endsWith } from 'lodash';
+
+import { protocols, customProtocols } from '../values/protocols';
 
 export default function parseURL(query) {
-  const trimmedQuery = trim(query);
-  const formattedQuery = trimmedQuery.includes('://') ? trimmedQuery : `nos://${trimmedQuery}`;
-  return new URL(formattedQuery);
+  const trimmedQuery = trimEnd(trim(query), '/').split('://').pop();
+
+  const tld = filter(customProtocols, (protocol) => endsWith(trimmedQuery, protocol.tld));
+  const tldToApply = isEmpty(tld) ? protocols.https : tld[0].protocol;
+
+  const url = trimmedQuery.includes('localhost:')
+    ? `${protocols.http}//${trimmedQuery}`
+    : `${tldToApply}//${trimmedQuery}`;
```

Comment:
Let's always forward to http.  Any server that enforces https will automatically redirect from http.

Here's an example of how Chrome handles no protocol being specified, you can see it always leads to http.  That seems safer since we can't assume that a server implements https.

![https-redirect](https://user-images.githubusercontent.com/169093/44629452-28e05d00-a915-11e8-9435-5950b00cff91.gif)

Reply:
@deanpress Should we force that upon the user?  I know most sites redirect from http to https these days, but if the user types in an address without the TLD and it doesn't support https, then the user will get a 404, which might be confusing.

Reply:
Yeahh, mostly with nginx they set a permanent redirect to `https`, but inside the client, if I enforce `http` and I go to google, it stays with `http` :P 
The downside is that you won't be able to visit any website over `http` as it's enforced in the client in this PR

Reply:
@DalderupMaurice I think it only looks like it stays http because of a bug with the address not updating on redirects, which I'm fixing right now.

Reply:
I'll mark it as blocked until the PR is up @mhuggins

Reply:
@mhuggins Good point. Let's stick with http and not enfore https within the client.